main.md

```diff
@@ -204,17 +209,41 @@ If `datatypes` and `actions` are used in combination, the permissions the client
       ],
       "datatypes": [
          "contacts",
-         "contracts"
+         "photos"
       ]
    }
 ]
 ```
-the client is requesting read and write access to the contacts and contracts belonging to customers in a customer information API.
-
-`identifier`:
-:   A string identifier indicating a specific resource available at the API. 
-
-An API MAY define its own extensions, subject to the `type` of the request. It is assumed that the full structure of each of the authorization data elements is tailored to the needs of a certain application, API, or resource type. The example structures shown above are based on certain kinds of APIs that can be found in the Open Banking space. 
+the client is requesting read and write access to both the contacts and photos belonging to customers in a customer information API. If the client wishes to have finer control over its access, it can send multiple objects. For example:
+```JSON
+[
+   {
+      "type": "customer_information",
+      "locations": [
+         "https://example.com/customers",
+      ]
+      "actions": [
+         "read"
+      ],
+      "datatypes": [
+         "contacts"
+      ]
+   },
+   {
+      "type": "customer_information",
+      "locations": [
+         "https://example.com/customers",
+      ]
+      "actions": [
+         "write"
+      ],
+      "datatypes": [
+         "photos"
+      ]
+   }
+]
+```
+The client is asking for read access to the contacts and write access to the photos in the same API endpoint.
 
 Note: Applications MUST ensure that their authorization data types do not collide. This is either achieved by using a namespace under the control of the entity defining the type name or by registering the type with the new `OAuth Authorization Data Type Registry` (see (#iana_considerations)). 
```

Comment:
The more text we add to this section the less prominent does this very important statement get. I suggest to move it up to the description of "type".

Reply:
See above, I think we can refactor this section to split `type` into its own discussion so that this warning is separate. I'd suggest we push this change set to meet the deadline and reshuffle this in the next draft, so we don't lose anything in our rush.

Reply:
ok. I will create an issue.